integration/integrate-with-image-vulnerability-scanners.adoc

```diff
@@ -32,22 +32,23 @@ This enhanced support gives you greater flexibility and choice in managing your
 
 You can set up {product-title-short} to obtain image vulnerability data from the following commercial container image vulnerability scanners:
 
-* {product-title-short} Scanner (recommended)
+* {product-title-short} Scanner V4: Beginning with release 4.4, a new scanner is introduced that is built on the {product-title-short} StackRox and Clair scanners. The new scanner, Scanner V4, supports scanning of language and OS-specific image components. You do not have to create an integration to use this scanner, but you must enable it during or after installation. You must use this scanner in conjunction with the StackRox Scanner. See the installation documentation for more information.
+* StackRox Scanner: This scanner is the default scanner in {product-title-short}. It originates from a fork of the Clair v2 open source scanner. You must continue using this scanner for RHCOS node scanning and for active vulnerability management until that functionality is supported using Scanner V4 in a future release.
+// provide instructions for selecting this scanner to use
 * link:https://github.com/quay/clair[Clair]
 * link:https://cloud.google.com/container-registry/docs/container-analysis[Google Container Analysis]
 * link:https://quay.io[Red Hat Quay]
 
 [IMPORTANT]
 ====
-{product-title-short} Scanner is the preferred image vulnerability scanner to use with {product-title-short}.
-For more information about scanning container images with {product-title-short} Scanner, see xref:../operating/examine-images-for-vulnerabilities.adoc#scanning-images_examine-images-for-vulnerabilities[Scanning images].
+The StackRox Scanner, in conjunction with Scanner V4 (optional), is the preferred image vulnerability scanner to use with {product-title-short}.
+For more information about scanning container images with the StackRox Scanner and Scanner V4, see xref:../operating/examine-images-for-vulnerabilities.adoc#scanning-images[Scanning images].
 ====
 
-If you use one of these products in your DevOps workflow, you can use the {product-title-short} portal to configure an integration with your vulnerability scanner. After the integration, the {product-title-short} portal shows the image vulnerabilities and you can triage them easily.
+If you use one of these alternative scanners in your DevOps workflow, you can use the {product-title-short} portal to configure an integration with your vulnerability scanner. After the integration, the {product-title-short} portal shows the image vulnerabilities and you can triage them easily.
```

Comment:
Perhaps we should let people know what happens when multiple scanners are configured.

In this scenario, we attempt to use the non-StackRox/ACS and Clair scanners first. If those fail, then we try to use a configured Clair. Then we try Scanner V4, if configured. Then we try StackRox Scanner (only when Scanner V4 is not configured). @dcaravel to confirm this <-

Reply:
Confirmed